docker/server/config/config-mysql.properties

```diff
@@ -1,6 +1,6 @@
 # Servers.
 conductor.jetty.server.enabled=true
```

Comment:
Is this property needed? I think its not even considered anymore.

Reply:
good catch, i still have to cleanup the property files in the docker directories, will add this in the next commit